mtv-migration/schemas/workflow-output-schema.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowResult",
+    "description": "Schema of workflow output",
+    "type": "object",
+    "properties": {
+        "result": {
+            "$ref": "../shared/schemas/workflow-result-schema.json",
```

Comment:
is this the correct path? I don't recall where we create the `shared` folder

Reply:
https://github.com/parodos-dev/serverless-workflows/blob/main/Makefile#L107 is making sure the shared folder is copied next to the workflow https://github.com/parodos-dev/serverless-workflows/blob/main/shared/schemas/workflow-result-schema.json